torchrl/objectives/ddpg.py

```diff
@@ -233,14 +236,15 @@ def _forward_value_estimator_keys(self, **kwargs) -> None:
         self._set_in_keys()
 
     def _set_in_keys(self):
-        keys = [
-            ("next", self.tensor_keys.reward),
-            ("next", self.tensor_keys.done),
+        in_keys = {
+            unravel_key(("next", self.tensor_keys.reward)),
+            unravel_key(("next", self.tensor_keys.done)),
```

Comment:
we should check but since it's a TensorDictModuleBase the in_keys are unravelled by default
https://github.com/pytorch-labs/tensordict/blob/accd8a4a31ec749f52e75a87a875424652069163/tensordict/nn/common.py#L474-L495

So I think we can spare the effort of doing that here

Reply:
but even if they are already unraveled, we are creating a new tuple `("next", already_unraveled_key)` which could be 
`("next","done")` or `("next",("nested", "done"))`

Reply:
that is why i am only unravleing the ones where we are putting next

Reply:
look at the link: they will be unravelled after you set them in theory

Reply:
funky stuff! i ll make sure to test it

Reply:
Ah maybe not for properties, this one's harder actually
Let's keep it as it is